### PR TITLE
Add transaction options

### PR DESCRIPTION
**Pull request by petergebala**
Its a fix for nested transactions: https://api.rubyonrails.org/classes/ActiveRecord/Transactions/ClassMethods.html#module-ActiveRecord::Transactions::ClassMethods-label-Nested+transactions

### CODE REVIEW

**Comment by petergebala** [comment]
We need specs for that

File: lib/opera/operation/instructions/executors/transaction.rb [hunk 1]
```diff
@@ -8,7 +8,8 @@ class Transaction < Executor
           class RollbackTransactionError < Opera::Error; end
 
           def call(instruction)
-            transaction_class.send(transaction_method) do
+            arguments = transaction_options.present? ? [transaction_method, transaction_options] : [transaction_method]
```
Comment (petergebala) [inline lib/opera/operation/instructions/executors/transaction.rb:11]: line is too long...

**Comment by MariaFamador** [comment]
Hi @petergebala, thanks for fixing this. Should I wait for specs before reviewing?

**Review by rrrodrigo — APPROVED** [review]
👍

**Review by kikorb — APPROVED** [review]

**Pull request merged**